client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -83,12 +83,22 @@ class ImageEditorCanvas extends Component {
 		}
 	}
 
-	getImage( url ) {
+	isBlobSrc( src ) {
+		return startsWith( src, 'blob' );
+	}
+
+	getImage( src ) {
 		const { onLoadError, mimeType } = this.props;
 
 		const req = new XMLHttpRequest();
-		req.open( 'GET', url + '?', true ); // Fix #7991 by forcing Safari to ignore cache and perform valid CORS request
+
+		if ( ! this.isBlobSrc( src ) ) {
+			src = src + '?'; // Fix #7991 by forcing Safari to ignore cache and perform valid CORS request
```

Comment:
I think we may want to make this a little smarter in case we load in url that may already have query parameters (for example width/height)